src/main/scala/RPS.scala

```diff
@@ -0,0 +1,27 @@
+package rps
+
+object RPS {
+
+  def rpsValue(choice: String): Int = choice.toLowerCase match {
+    case "rock" => 1
+    case "paper" => 2
+    case "scissors" => 3
+    case _ => -1
+  }
+
+  def rpsChoiceFromVal(choiceValue: Int): String = choiceValue match {
+    case 1 => "rock"
+    case 2 => "paper"
+    case 0 | 3 => "scissors"
```

Comment:
why do you need "0 | 3" does it not make scissors happen more if you just use the random number on it. also seeing that you use the rpsValue function to get the numbers is't having a case for 0 redundant?